server/url_params.go

```diff
@@ -50,7 +50,7 @@ func ExtractParametersFromUrl(url *url.URL, allBuilders []string) (params URLPar
 		}
 		for _, hint := range hintQuery {
 			// valid hints are: "hash", "contract_address", "function_selector", "logs", "calldata"
-			if hint != "hash" && hint != "contract_address" && hint != "function_selector" && hint != "logs" && hint != "calldata" {
+			if hint != "hash" && hint != "contract_address" && hint != "function_selector" && hint != "logs" && hint != "calldata" && hint != "default_logs" {
```

Comment:
eventually we should just use enums or something cause this conditional is pretty jank but not worth blocking this PR over it